### PR TITLE
PLAT-11870: Prepare GA release

### PR DESCRIPTION
**Pull request by symphony-soufiane**
### Description
Added a log of wdk version using Gradle resource filtering. A placehoder is added in spring application.yaml file that is replaced after the build with the current version.
Removed a disclaimer from README.

### Dependencies
N/A

### Checklist
- [x] Referenced a ticket in the PR title or description
- [x] Filled properly the description and dependencies, if any
- [ ] Unit/Integration tests updated or added
- [ ] Javadoc added or updated
- [ ] Updated the documentation in [docs folder](../docs)

### CODE REVIEW

**Comment by symphony-youri** [comment]
I'd we usually don't reference our internal ticker numbers in OSS projects as they don't mean much for external contributors.
What about using https://docs.spring.io/spring-boot/docs/current/gradle-plugin/reference/htmlsingle/#integrating-with-actuator.build-info  and then injecting https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/info/BuildProperties.html?

File: workflow-bot-app/build.gradle [hunk 1]
```diff
@@ -82,3 +84,9 @@ publishing {
         }
     }
 }
+
+processResources {
```
Comment (symphony-youri) [inline workflow-bot-app/build.gradle:88]: actually


processResources {
    filesMatching('application.yaml') {
        expand(project.properties)
    }
}

could be used to also set:


bdk:
  default-headers:
    User-Agent: "Symphony-WDK/${project.version} Java/\${java.version}"

in the default application yaml

Comment (symphony-youri) [reply workflow-bot-app/build.gradle:88]: our yours is simpler to make it work, no need to escape

**Comment by symphony-soufiane** [comment]
ok to test

**Review by symphony-youri — APPROVED** [review]

**Pull request merged**